Submitted By: Ken Moffat (ken at linuxfromscratch dot org)
Origin: fedora
        Rediffed against 4.1.0 by Chris Staub
	Rediffed against 4.2.0 by Joe Ciccone
	Rediffed agaisnt 4.2.4 by Ken Moffat
Date: 2005-11-22
Initial package version: 4.0.2
Description: In a native ppc64 build, this fixes a linkage failure while
installing a 32-bit nof/ (no -hardware- floating point) library, and also gets
rid of 1500 failures in the g++ testsuite.  From fedora gcc-4.0.1, rediffed to
apply with -p1.

diff -Naur gcc-4.2.4.orig/gcc/config/rs6000/t-linux64 gcc-4.2.4/gcc/config/rs6000/t-linux64
--- gcc-4.2.4.orig/gcc/config/rs6000/t-linux64	2008-03-05 17:25:28.000000000 +0000
+++ gcc-4.2.4/gcc/config/rs6000/t-linux64	2008-06-12 20:45:36.000000000 +0100
@@ -14,13 +14,13 @@
 # it doesn't tell anything about the 32bit libraries on those systems.  Set
 # MULTILIB_OSDIRNAMES according to what is found on the target.
 
-MULTILIB_OPTIONS        = m64/m32 msoft-float
-MULTILIB_DIRNAMES       = 64 32 nof
+MULTILIB_OPTIONS        = m64/m32
+MULTILIB_DIRNAMES       = 64 32
 MULTILIB_EXTRA_OPTS     = fPIC mstrict-align
-MULTILIB_EXCEPTIONS     = m64/msoft-float
-MULTILIB_EXCLUSIONS     = m64/!m32/msoft-float
-MULTILIB_OSDIRNAMES	= ../lib64 $(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib) nof
-MULTILIB_MATCHES        = $(MULTILIB_MATCHES_FLOAT)
+MULTILIB_EXCEPTIONS     =
+MULTILIB_EXCLUSIONS     =
+MULTILIB_OSDIRNAMES	= ../lib64 $(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib)
+MULTILIB_MATCHES        =
 
 softfp_wrap_start := '\#ifndef __powerpc64__'
 softfp_wrap_end := '\#endif'
